packages/oauth/package.json

```
@@ -54,7 +50,6 @@
     "commands": "./lib/commands",
```

core doesn't cover short flags for help and version. This re-adds the functionality.
```suggestion
    "additionalHelpFlags": ["-h"],
    "additionalVersionFlags": ["-v"],
    "commands": "./lib/commands",
```